Source/WebCore/html/PermissionsPolicy.cpp

```diff
@@ -415,6 +444,8 @@ PermissionsPolicy::PermissionsPolicy(const Document& document)
         if (computeInheritedPolicyValueInContainer(feature, ownerPermissionsPolicy, document.securityOrigin().data()))
             m_inheritedPolicy.add(feature);
     });
+    if (ownerPermissionsPolicy)
+        m_policyDirective = WTFMove(ownerPermissionsPolicy->containerPolicy);
```

Comment:
This function is implemented based on spec https://w3c.github.io/webappsec-permissions-policy/#algo-create-for-navigable. It seems not likely `m_policyDirective` is needed if other features do not require it.